pandas/io/tests/json/test_json_norm.py

```diff
@@ -63,6 +63,21 @@ def test_simple_normalize(self):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_simple_normalize_with_default_separator(self):
```

Comment:
can you add the issue number as a comment

you can make this all 1 test (with 3 sub-tests).